packages/jsii-pacmak/test/generated-code/__snapshots__/target-java.test.js.snap

```diff
@@ -27956,6 +27963,84 @@ exports[`Generated code for "jsii-calc": <runtime-type-check-diff>/java/src/main
  
      /**
       */
+@@ -249,35 +284,35 @@
+     }
+ 
+     /**
+      */
+     @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Stable)
+-    public void setUnionProperty(final @org.jetbrains.annotations.NotNull java.lang.String value) {
++    public void setUnionProperty(final @org.jetbrains.annotations.NotNull java.lang.String value) throws IllegalArgumentException {
```

Comment:
I know that the code gen changed how this is rendered; so, is this expected? There's a `throws` declaration, but no exception thrown here.